t/10-jobs.t

```diff
@@ -689,13 +690,44 @@ subtest 'carry over, including soft-fails' => sub {
             $job_info = $t->app->minion->jobs({tasks => ['hook_script']})->next;
             is $job_info->{state}, 'inactive', 'hook script has been retried with long delay';
             is_deeply($job_info->{args}[2],
-                {delay => 60, retries => 1440, skip_rc => 142, kill_timeout => '30s', timeout => '5m'});
+                {delay => 60, retries => $default_retries, skip_rc => 142, kill_timeout => '30s', timeout => '5m'});
             $notes = $job_info->{notes};
             is($notes->{hook_cmd}, 'echo delayed && exit 142;', 'real hook cmd in notes if result matches (5)');
             like($notes->{hook_result}, qr/delayed/, 'real hook cmd from config called if result matches (5)');
             is $notes->{hook_rc}, 142, 'exit code of the hook cmd is as expected';
             is $job_info->{retries}, 1, 'hook script has been retried once because of delay';
         };
+
+        subtest 'There is a linear delay for each hook script which exits with 142' => sub {
```

Comment:
I agree with @perlpunk. To make this shorter you can e.g. remove the "There is a" part which is not adding anything useful and rephrase it a little bit:
```suggestion
        subtest 'linear delay of each hook scripts exiting with 142' => sub {
```